Add roomba total cleaned area sensor (#30559)
home-assistant/home-assistant.github.io: source/_integrations/roomba.markdown

- **Total missions**: The total number of all missions
- **Scrubs**: Total number of times the robot has executed a "scrub"
- **Total cleaning time**: How many hours the robot has spent cleaning in total

> - **Total cleaned area**: The total amount of m² that Roomba has cleaned

Reviewer: ```suggestion
- **Total cleaned area**: The total area in m² the robot has cleaned
```
Naming of the robot should be consistent with previous bullets.

Will the units be subject to localisation e.g. square feet? If so maybe omit m^2 here.

Author: I don't think localisation of this sensor will be possible in near future. See: https://github.com/home-assistant/architecture/discussions/849

Reviewer: Ok, makes sense.
